docs/middleware.md

```diff
@@ -118,6 +118,17 @@ schema.pre('save', function(next) {
 });
 ```
 
+Here are the possible strings that can be passed to `pre()`
```

Comment:
You're missing a whole bunch. `estimatedDocumentCount`, `findOneAndReplace`, `aggregate`, `insertMany`, and more.

Also, please put this list immediately after "* [insertMany](api/model.html#model_Model-insertMany)" on L69